Notifications/Notifications/Presentation/Settings/NotificationsSettingsView.swift

```diff
@@ -64,10 +63,19 @@ public struct NotificationsSettingsView: View {
                                 .toggleStyle(SwitchToggleStyle(tint: Theme.Colors.toggleSwitchColor))
                                 .frame(width: 50)
                                 .accessibilityIdentifier("discussion_switch")
-                                .onTapGesture {
-                                    viewModel.toggleNotificationsPermissionAction()
-                                }
-                            
+                                .simultaneousGesture(
+                                    TapGesture().onEnded {
+                                        Task {
+                                            await viewModel.toggleNotificationsPermissionAction()
+                                        }
+                                    }
+                                )
+                                .simultaneousGesture(
+                                    DragGesture(minimumDistance: 20, coordinateSpace: .local).onEnded { _ in
+                                        Task {
+                                            await viewModel.toggleNotificationsPermissionAction()
+                                        }
+                                    })
```

Comment:
It appears there is an issue with the toggles; if I long-press the switch, it turns on or off without making any API calls.